plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java

```diff
@@ -199,7 +199,8 @@ protected void assertOverwritePartition(String testTable)
                         .row(2L)
                         .build());
 
-        query(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable))
+        computeActual(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable));
+        query(format("SELECT count(*) FROM %s WHERE regionkey = 5", testTable))
```

Comment:
That is ok - though I would prefer asserting on actual rows, not count.

Reply:
Added second commit to switch to exact rows.